lib/ui/home/HomeDailyIlliniWidget.dart

```diff
@@ -360,14 +368,14 @@ class _MainStoryWidget extends StatelessWidget {
   }
   Widget _buildImage() {
     return StringUtils.isNotEmpty(illiniItem?.thumbImageUrl)
-        ? ModalImageHolder(child: Image.network(illiniItem!.thumbImageUrl!, excludeFromSemantics: true, loadingBuilder: (context, child, loadingProgress) {
+        ? Image.network(illiniItem!.thumbImageUrl!, excludeFromSemantics: true, loadingBuilder: (context, child, loadingProgress) {
```

Comment:
Please use `ModalImageHolder` instead of `Image.network`. That's an requirement since a long time where we show bigger image when user taps on it.

Reply:
fixed, tapping image now shows bigger image